spec/README.md

```diff
@@ -141,6 +143,19 @@ subject and unambiguously identifying the types of the [predicate]. It is a
 >
 > MUST be non-empty and unique within `subject`.
 
+`subject[*].uri` _string ([ResourceURI]), optional_
+
+> Identifies an artifact by an immutable resource URI.
+> The `uri` field MUST be specified only if a subject cannot have a `digest` field.
+> 
+> Examples of subjects that cannot be identified by a `digest`, but that can be
+> identiified by an `uri` are a source-code revision (URI = svn+ssh://host/path/revision-number)
```

Comment:
nit: type should be "identified"

Reply:
Should I replace the two instances of "cannot have a digest" with "cannot be identified by a digest"? The word"type" does not appear in the diff.

Reply:
I have replaced "identiified" with "identified".